src/Helper/Scripts.php

```diff
@@ -44,6 +44,39 @@ public function add($src, $pos = 100)
 
     /**
      *
+     * Adds a <script> tag to the stack with additional attributes
+     *
+     * @param string $src The source href for the script.
+     *
+     * @param string $attr The additional attributes
+     *
+     * @param int $pos The script position in the stack.
+     *
+     * @return null
+     *
+     */
+    public function addAttr($src, array $attr = null, $pos = 100)
```

Comment:
Hey,

I guess adding `$attr = array()` will not break anything on `add` . Do you still think it is a BC break ?